include/El/blas_like/level1/EntrywiseMap.hpp

```diff
@@ -160,7 +160,6 @@ void EntrywiseMap
     B.remoteVals_.resize( numRemoteEntries );
     for( Int k=0; k<numRemoteEntries; ++k )
         B.remoteVals_[k] = func(A.remoteVals_[k]);
-    B.multMeta = A.multMeta;
```

Comment:
I think because a few lines early we issue:

`  B.distGraph_ = A.distGraph_;`

which assigns the multMeta objects.